service/application/pom.xml

```diff
@@ -387,6 +413,65 @@
+                <artifactId>build-helper-maven-plugin</artifactId>
+                <version>3.6.0</version>
+                <executions>
+                    <execution>
+                        <id>add-source</id>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>add-source</goal>
+                        </goals>
+                        <configuration>
+                            <sources>
+                                <source>${project.build.directory}/generated-sources/swagger/src/main/java</source>
+                            </sources>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>add-test-source</id>
+                        <phase>generate-test-sources</phase>
+                        <goals>
+                            <goal>add-test-source</goal>
+                        </goals>
+                        <configuration>
+                            <sources>
+                                <source>${project.build.directory}/generated-sources/swagger/src/test/java</source>
+                            </sources>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>io.swagger</groupId>
+                <artifactId>swagger-codegen-maven-plugin</artifactId>
+                <version>${swagger-codegen-maven-plugin.version}</version>
+                <executions>
+                    <execution>
+                        <id>sempv2-vmr-9.12</id>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>generate</goal>
+                        </goals>
+                        <configuration>
+                            <inputSpec>${project.basedir}/src/main/resources/oas/semp-v2-swagger-config.json</inputSpec>
+                            <language>java</language>
+                            <output>${project.build.directory}/generated-sources/swagger</output>
+                            <invokerPackage>com.solace.client.sempv2</invokerPackage>
+                            <apiPackage>com.solace.client.sempv2.api</apiPackage>
+                            <modelPackage>com.solace.client.sempv2.model</modelPackage>
+                            <configOptions>
+                                <dateLibrary>java8</dateLibrary>
+                                <hideGenerationTimestamp>true</hideGenerationTimestamp>
+                                <sourceFolder>src/main/java</sourceFolder>
+                                <testSourceFolder>src/test/java</testSourceFolder>
+                            </configOptions>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
```

Comment:
I had to add some exclusions on jacoco reports on [ep-core PR](https://github.com/SolaceDev/maas-ep-core/pull/2916/files#diff-e111dd0e82f440d2675d1ad5b10fe3ffde1c976ac42f203065af98550372b13eR417) as well. Please make sure we will pass the line coverage requirements before merging this PR.